### ISSUE TITLE
Document rule exclusion pkg to plugin migration from CRS3 to CRS4 in separate blog post

### ISSUE THREAD

**Issue by dune73** [coreruleset/coreruleset#3565]
No need to rush this before the CRS4 release. But it's something to accompany the documentation.

**Comment by fzipi** [comment]
@dune73 Do we want to do this in June? Or is there a timeline?

**Comment by dune73** [comment]
That would be an alternative to the idea to describe how to create your own plugin. And actually quite a good alternative. What do you think?

**Comment by dune73** [comment]
Ping

**Comment by dune73** [comment]
Ping @fzipi.

**Comment by fzipi360** [comment]
Yes, makes sense. How to create your own plugin, and then mentioning that this is how we migrated things. 👍

**Comment by fzipi** [comment]
Yes, makes sense. How to create your own plugin, and then mentioning that this is how we migrated things. 👍

**Comment by dune73** [comment]
OK. Consider this done!

**Comment by fzipi** [comment]
Do you want to keep this issue here or transfer to the webpage repo?

**Comment by dune73** [comment]
I'm all for migration to webpage.

As documented above I had planned to write this for the WAF month media campaign by OWASP's outreach coordinator Jason. Then he left and I did not write this.
